From 867924e30ae7457589a2b03cdf87eb64b63f9d0a Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Mon, 9 Jan 2017 17:49:25 +0800
Subject: [PATCH 1494/5242] MLK-13733-1 ARM: dts: imx7ulp: add rpmsg instance
 for power management

commit  87b3043258423bf02a719e7b93ee2e65a50984b7 from
https://source.codeaurora.org/external/imx/linux-imx.git

On i.MX7ULP, add a new RPMSG instance for power management.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp-evk.dts |    9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/imx7ulp-evk.dts b/arch/arm/boot/dts/imx7ulp-evk.dts
index 9d2d2c5..1202c79 100644
--- a/arch/arm/boot/dts/imx7ulp-evk.dts
+++ b/arch/arm/boot/dts/imx7ulp-evk.dts
@@ -418,11 +418,12 @@
 &rpmsg{
 	/*
 	 * 64K for one rpmsg instance, default using 2 rpmsg instances:
-	 * --0x9FF00000~0x9FF01FFF: pingpong
-	 * --0x9FF10000~0x9FF11FFF: pmic
+	 * --0x9FF00000~0x9FF0FFFF: pingpong
+	 * --0x9FF10000~0x9FF1FFFF: pmic
+	 * --0x9FF20000~0x9FF2FFFF: pm
 	 */
-	vdev-nums = <2>;
-	reg = <0x9FF00000 0x20000>;
+	vdev-nums = <3>;
+	reg = <0x9FF00000 0x30000>;
 	status = "okay";
 };
 
-- 
1.7.9.5

